ReSwift/CoreTypes/Store.swift

```diff
@@ -155,3 +186,10 @@ open class Store<State: StateType>: StoreType {
         _ actionCreatorCallback: @escaping ((ActionCreator) -> Void)
     ) -> Void
 }
+
+extension Store where State: Equatable {
+    open func subscribe<S: StoreSubscriber>(_ subscriber: S)
+        where State: Equatable, S.StoreSubscriberStateType == State {
```

Comment:
isn't `where State: Equatable` redundant to line 190? Looking at it, I wonder if we should move the method up into the main body with the type requirement specifier directly.